x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/RunAnalyticsAction.java

```diff
@@ -23,7 +23,7 @@
 public class RunAnalyticsAction extends Action<AcknowledgedResponse> {
 
     public static final RunAnalyticsAction INSTANCE = new RunAnalyticsAction();
-    public static final String NAME = "cluster:admin/xpack/ml/analytics/run";
+    public static final String NAME = "cluster:admin/xpack/ml/dataframe/run";
```

Comment:
-> `data_frame`

Reply:
Sorry, I should have deleted this class entirely.